cypress/integration/shared/clusterConfiguration.ts

```diff
@@ -78,6 +83,41 @@ export const waitForHostTablePopulation = (
   });
 };
 
+export const waitForHostsSubnet = (cy) => {
+  // wait until hosts subnet populated in the cluster details
+  cy.get('#form-input-hostSubnet-field')
+    .find('option', { timeout: HOST_DISCOVERY_TIMEOUT })
+    .should(($els) => {
+      expect($els.length).to.be.gt(0);
+    })
+    .and(($els) => {
+      expect($els[0]).not.to.have.text('No subnets available');
+    });
+};
+
+export const waitForHostsToBeReady = (cy, numMasters = NUM_MASTERS, numWorkers = NUM_WORKERS) => {
```

Comment:
`cy` is missing type
`numMasters` type can be automatically deduced from the initial value (means `number` in this case)

Reply:
Done